home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/service/poll/PollServiceImpl.java

```diff
@@ -89,7 +91,7 @@ public PollDto update(Long cooperationId, Long id, PollDto pollDto) {
             .filter(poll1 -> poll1.getCooperation().getId().equals(cooperationId))
             .orElseThrow(() -> new NotFoundHomeException(String.format(NOT_FOUND_MESSAGE, "Poll", id)));
         validatePollStatus(poll, pollDto.getStatus());
-        validateCreationDate(poll);
+        validateCreationDate(poll.getCreationDate(), "Poll has already started");
```

Comment:
It's seems like logic violation here. You check creation date of poll which already stored in DB. This validation already passed during poll creation in line 73. Consider validation of pollDto which provided in parameters and matching creationDate of pollDto and poll which loads from DB in line 90